src/js/components/MultiSelect/MultiSelect.js

```diff
@@ -0,0 +1,680 @@
+                      );
+                      if (nextFocus) nextFocus.focus();
+                      const result = allOptions.find(
+                        (obj, j) => optionValue(j) === intermediate[index - 1],
+                      );
+                      setShowA11yDiv(
+                        `Unselected ${iLabel}. Focus moved to 
+                                    ${getOptionLabel(
+                                      allOptions.indexOf(result),
+                                      allOptions,
+                                      labelKey,
+                                    )}`,
+                      );
+                    } else if (dropButtonRef.current)
+                      dropButtonRef.current.focus();
+                  }
+                }
+              }}
+              key={iLabel}
+              id={`selected-${iValue}`}
+            >
+              {children && child ? (
+                child
+              ) : (
+                <CheckBox
+                  disabled={isDisabled(indexOptions)}
+                  label={
+                    <Box alignSelf="center" width="100%" align="start">
+                      {iLabel}
+                    </Box>
+                  }
+                  key={iLabel}
+                  pad="xsmall"
+                  tabIndex="-1"
+                  checked={value.includes(iValue)}
+                />
+              )}
+            </SelectOption>
+          );
+        }
+        return undefined;
+      },
+      [
+        valueKey,
+        allOptions,
+        children,
+        dropButtonRef,
+        isDisabled,
+        labelKey,
+        onSelectChange,
+        optionValue,
+        value,
+      ],
+    );
+
+    // element to show, trumps inputValue
+    const selectValue = useMemo(() => {
+      if (valueLabel instanceof Function) {
+        if (value) return valueLabel(value);
+      } else if (valueLabel) return valueLabel;
+      else if (value.length > 0 && visibleSelection) {
+        return (
+          <>
+            <Box
+              width="100%"
+              role="listbox"
+              aria-multiselectable
+              a11yTitle="Selected Options"
+            >
+              {value &&
+                allOptions
+                  .filter(
+                    (i) =>
+                      value.indexOf(
+                        valueKey && valueKey.reduce ? applyKey(i, valueKey) : i,
+                      ) !== -1,
+                  )
+                  /* eslint-disable-next-line array-callback-return, 
+                  consistent-return */
+                  .map((i) => visibleValue(i))}
+              {showA11yDiv && (
+                <Box
+                  height="0px"
+                  width="0px"
+                  overflow="hidden"
+                  // announce when an item is removed from selected options
+                  aria-live="assertive"
+                >
+                  <Text>{showA11yDiv}</Text>
```

Comment:
If this is just for screenreader, can we remove the `Text`?
```suggestion
                  {showA11yDiv}
```

Reply:
Yes, removed `Text` in latest commit